### ISSUE TITLE
Attempting to encode from Windows hosted SMB network share completes instantly with no output.

### ISSUE THREAD

**Issue by TorazChryx** [RandomEngy/VidCoder#835]
So this has me scratching my head, I've got a windows machine full of drives acting as a NAS, and a big beefy desktop machine I'd like to use to transcode some video (as a 5950X is rather more cpu than the Sandy Bridge quadcore in the NAS box), but.. well, something is broken. 

Batch queue a folder of files up for H264 -> H265 transcode, set it to either drop in a local folder on the transcoding machine OR replace the source files upon completion and the same thing happens, the "transcode" takes about a second and it flags as complete. But there's no output, nor could there be in that timeframe.

Whisky Tango Foxtrot.

**Comment by RandomEngy** [comment]
What are the source file paths? What's in the encode logs?

**Comment by TorazChryx** [comment]
Good questions both, it didn't even occur to me that there was logging (and actually there wasn't, as I hadn't turned on logs)

So I just punted a single file at it from the same network share (which is mapped to a drive letter, so it's z:\<folder>\<subfolder>\) and THAT'S transcoding successfully... so I'm going to have to try a batch now to get a log that actually says something useful.

*a short time later*

Okay... so I gave it one folder among many subfolders to batch transcode, and THAT'S seemingly working correctly, ETA for the whole folder is like 50 minutes... I'm going to cancel that and try adding the whole folder structure (which worked perfectly when I did it on the NAS directly, it's just... 2600K, which was also the estimated number of minutes needed to complete the task.)

Before seeing the logs (because obviously they're gonna take a little while to generate whilst I'm writing this near stream of consciousness post) I find myself wondering if the issue is multiple nested folders containing mkv/mp4 content underneath the path over SMB, as opposed to one singular folder that is flat in structure. spitballing here though, might be completely off the mark.

Um... okay, it now appears to be working perfectly fine.
I'm somewhat perplexed at this juncture, although I DID come into this particular instance of vidcoder 6.33 through a different sequence of actions, that being that I pointed it at a singular file initially, which I had not done on the previous attempts.

Consider the matter closed for the moment I guess, next time I restart it I'll try the same approach and see how it behaves, and if it acts up again I'll be back into this thread with a log that actually shows the error. (of course, there's bonus points if turning on the logging is somehow whats fixed it.)
